subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -1849,142 +1849,13 @@ uint8_t common_tx_buffer_alloc_count(void)
 }
 #endif /* LLCP_TX_CTRL_BUF_QUEUE_ENABLE */
 
-void test_int_mem_proc_ctx(void)
+struct proc_ctx *pub_proc_ctx_acquire(void)
```

Comment:
oh, there was this one - why the 'pub' prefix here and the other places?

Reply:
so that we can expose some of the internal API for unit testing. pub is short for public

Reply:
and replaced the prefix with llcp_